### ISSUE TITLE
With v3.0.0 mpl_toolkits.axes_grid1.make_axes_locatable().append_axes breaks in Jupyter

### ISSUE THREAD

**Issue by schiotz** [matplotlib/matplotlib#12657]
### Bug report

**Bug summary**

After upgrading to 3.0.0, colormaps added to images with mpl_toolkits.axes_grid1.make_axes_locatable().append_axes overwrite the entire image.  This happens in a jupyter notebook with %matplotlib inline but not with %matplotlib notebook.  Also sometimes seen in scripts saving to .PNG (but I cannot reproduce it today).

**Code for reproduction**

This should be run as a Jupyter Notebook so see the problem.

```python
import matplotlib.pyplot as plt
from mpl_toolkits.axes_grid1 import make_axes_locatable
import numpy as np

a = np.random.normal(size=(300,300))
cmap = 'hot'

fig, ax = plt.subplots(figsize=(6,5))
imshow = ax.imshow(a,cmap=cmap)
divider = make_axes_locatable(ax)
cax = divider.append_axes("right", size="5%", pad=0.05)
plt.colorbar(imshow, cax=cax)
```

**Actual outcome**

The colorbar overwrites the entire image.

**Expected outcome**

A nice colorbar next to the image - it worked in v. 2.1.1 and I am 99% sure it worked in later 2.X versions of matplotlib.

**Matplotlib version**

  * Operating system:  MacOS 10.13.6  AND Ubuntu 18.04.1
  * Matplotlib version: 3.0.0
  * Matplotlib backend (`print(matplotlib.get_backend())`):  module://ipykernel.pylab.backend_inline
  * Python version: 3.7.0 and 3.6.6
  * Jupyter version (if applicable): notebook 5.7.0
  * Other libraries:

**Comment by schiotz** [comment]
PDF showing the problem:

[TestMatplotlib.pdf](https://github.com/matplotlib/matplotlib/files/2522988/TestMatplotlib.pdf)

**Comment by jklymak** [comment]
Yes this is a duplicate.  Sorry this broke but should be fixed in 3.0.1

**Comment by schiotz** [comment]
@jklymak Thank you very much for your very quick response, and that on a Sunday!
